lib/run.go

```diff
@@ -220,7 +223,8 @@ func genDeplist(acipath string, reg registry.Registry) ([]string, error) {
 			return nil, err
 		}
 
-		subdeps, err := genDeplist(path.Join(reg.Scratchpath, depkey), reg)
+		subdeps, err := genDeplist(
```

Comment:
Eh, I really don't care for the line wrapping, typically in Go we only wrap comments or some egregiously long lines